docs/download.md

```diff
@@ -41,40 +41,18 @@ for your hardware's minimum driver version.
 *For Cloudera and EMR support, please refer to the
 [Distributions](./FAQ.md#which-distributions-are-supported) section of the FAQ.
 
-### Download v22.12.0
-* Download the [RAPIDS
-  Accelerator for Apache Spark 22.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.12.0/rapids-4-spark_2.12-22.12.0.jar)
-
-This package is built against CUDA 11.5 and all CUDA 11.x versions are supported through [CUDA forward
-compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html). It is tested
-on V100, T4, A2, A10, A30 and A100 GPUs with CUDA 11.0-11.5.  For those using other types of GPUs which
-do not have CUDA forward compatibility (for example, GeForce), CUDA 11.5 or later is required. Users will
-need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
-
-### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 22.12.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.12.0/rapids-4-spark_2.12-22.12.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 22.12.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.12.0/rapids-4-spark_2.12-22.12.0.jar.asc)
-* Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
-* Import the public key: `gpg --import PUB_KEY`
-* Verify the signature: `gpg --verify rapids-4-spark_2.12-22.12.0.jar.asc rapids-4-spark_2.12-22.12.0.jar`
-
-The output if signature verify:
-
-	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
-
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Zstandard compression for Parquet and ORC
-* Support for Hive text reading
-* Improved performance on `like` operations
-* Tiered projections for more expressions to optimize performance  
-* Support for mapInArrow, instr and array_remove operations
-* z-ordering capability on Databricks Delta Lake
-* Dynamic Partition Pruning (DPP) on Databricks 
+* Delta Lake MERGE/DELETE/UPDATE 
```

Comment:
There's no PR merged for these.  MERGE is being worked on now, but it's not a given it will make the release.  DELETE and UPDATE will not.

Reply:
@jlowe  This is based on PRD which may have potential 23.02 items. I will try to keep the on-track ones, and wait for it to be merged. For now, let me remove DELETE/UPDATE from 23.02 release note.

Reply:
Sure I will keep this PR open and wait until end of the release.

Reply:
Updated right before end of the release. @jlowe  can you help take a look again?thx

Reply:
@viadea updated the release notes for Delta Lake merge/update/delete is experimental and can be enabled by a config.